packages/web_benchmarks/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: web_benchmarks
+description: A benchmark harness for performance-testing Flutter apps in Chrome.
+version: 0.0.1
+homepage: https://github.com/flutter/packages/tree/master/packages/web_benchmarks
+
+environment:
+  sdk: ">=2.7.0 <3.0.0"
+  flutter: ">=1.17.0 <2.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_test:
+    sdk: flutter
+  logging: ">=0.11.4 <1.0.0"
+  meta: ">=1.0.0 <2.0.0"
+  path: ">=1.7.0 <2.0.0"
+  process: ">=3.0.13 <4.0.0"
```

Comment:
Is it possible to update `process` to major version 4?

Gallery depends on `flutter_driver` from SDK, which depends on `process 4.0.0-nullsafety.2`; when I add `web_benchmarks` as a dependency, there's a conflict.

Reply:
Widened it to "<5.0.0". Does that work?

Reply:
Yes, it works. Thank you!